azure/templates/deploy-namespaced-service.yml

```diff
@@ -53,7 +53,7 @@ steps:
       export SERVICE_NAME="$(SERVICE_NAME)-$(APIGEE_ENVIRONMENT)-$(_NAMESPACE)"
       export SERVICE_BASE_PATH="$(SERVICE_BASE_PATH)"
       export APIGEE_ACCESS_TOKEN="$(secret.AccessToken)"
-      export PRODUCT_DISPLAY_NAME="Identity Service"
+      export PRODUCT_DISPLAY_NAME="Identity Service $(_NAMESPACE)"
```

Comment:
Previously when this was implemented there was a map to translate between the namespace 'short codes' (e.g. `int`) and more human-readable names (e.g. `Integration Environment`) and this was used to populate the product display name